### ISSUE TITLE
Git Version

### ISSUE THREAD

**Issue by branrickman** [jwiegley/git-from-the-bottom-up#19]
Hi there. I'm working through your book, and I notice that the last commit was a few years ago. My first question is what version is the last covered in this book. The second is, for someone who is trying to move from git dilettante to a working knowledge of git, are there any major changes in the last few years that are important for this level of understanding?

Thanks for all the work that has clearly gone into this guide- and for open sourcing it.

**Comment by ichn-hu** [comment]
I think the version covered in the book is Git 1.5.4.5, and since it introduced the most fundamental essentials of git, the content should be valid still for understanding.

**Comment by xim** [comment]
Yeah, there are some useful additions that might have been covered, like `git stash -p` or whatever, but it's still a useful resource.
